spec/std/http/server/handlers/static_file_handler_spec.cr

```diff
@@ -1,50 +1,50 @@
-require "spec"
+require "../../../spec_helper"
 require "http/server"
 
 private def handle(request, fallthrough = true, directory_listing = true, ignore_body = false)
   io = IO::Memory.new
   response = HTTP::Server::Response.new(io)
   context = HTTP::Server::Context.new(request, response)
-  handler = HTTP::StaticFileHandler.new "#{__DIR__}/static", fallthrough, directory_listing
+  handler = HTTP::StaticFileHandler.new datapath("static_file_handler"), fallthrough, directory_listing
   handler.call context
   response.close
   io.rewind
   HTTP::Client::Response.from_io(io, ignore_body)
 end
 
 describe HTTP::StaticFileHandler do
-  file_text = File.read "#{__DIR__}/static/test.txt"
+  file_text = File.read datapath("static_file_handler", "test.txt")
```

Comment:
maybe memorize `datapath("static_file_handler", "test.txt")` in a `static_file_path` variable for all the specs below that uses it?